templates/cockpit-apache-ssl.conf.erb

```diff
@@ -0,0 +1,14 @@
+### File managed with puppet ###
+
+<Location /webcon>
+  LoadModule proxy_wstunnel_module modules/mod_proxy_wstunnel.so
```

Comment:
Please don't use LoadModule this way. In the class you should `include apache::mod::proxy_wstunnel`.

Reply:
Updated